drivers/include/ng_at86rf2xx.h

```diff
@@ -137,6 +150,7 @@ typedef struct {
     uint8_t seq_nr;                     /**< sequence number to use next */
     uint8_t frame_len;                  /**< length of the current TX frame */
     uint16_t pan;                       /**< currently used PAN ID */
+    uint8_t chan;                       /**< currently used channel */
```

Comment:
can you put `ifdef` around this value, so the device descriptor does not grow in size for the other devices of the family?